examples/examples/multisig.rs

```diff
@@ -0,0 +1,65 @@
+// Copyright 2019-2022 Parity Technologies (UK) Ltd.
+// This file is dual-licensed as Apache-2.0 or GPL-3.0.
+// see LICENSE for license details.
+
+//! To run this example, a local polkadot node should be running. Example verified against substrate latest-38a955ba4a0.
+//!
+//! E.g.
+//! ```bash
+//! curl "https://releases.parity.io/substrate/x86_64-debian:stretch/latest/substrate/substrate" --output /usr/local/bin/substrate --location
+//! substrate --dev --tmp
+//! ```
+
+use sp_keyring::AccountKeyring;
+use subxt::{
+    tx::PairSigner,
+    OnlineClient,
+    SubstrateConfig,
+};
+
+#[subxt::subxt(runtime_metadata_path = "../artifacts/substrate_metadata.scale")]
+pub mod substrate {}
+
+#[tokio::main]
+async fn main() -> Result<(), Box<dyn std::error::Error>> {
+    tracing_subscriber::fmt::init();
+
+    let signer = PairSigner::new(AccountKeyring::Alice.pair());
+
+    // Create a client to use:
+    let api = OnlineClient::<SubstrateConfig>::new().await?;
+
+    // The call data for the `Multisig::as_multi` extrinsic.
+    // This must be of type `RuntimeCall` and represents the `Assets::freeze_asset` method.
+    // Otherwise, could have been obtained using `substrate::tx().assets().freeze_asset(100)`
+    let call = substrate::runtime_types::kitchensink_runtime::RuntimeCall::Assets(
+        substrate::runtime_types::pallet_assets::pallet::Call::freeze_asset { id: 100 },
+    );
```

Comment:
what about:

```rust
substrate::tx().assets().freeze_asset(100).call_data().clone()
```

does that work instead?

And if so, perhaps we could add an `into_call_data()` to mirror the `into_value() I added on the dynamic side, and avoid the clone here?